src/App.tsx

```diff
@@ -1,39 +1,54 @@
-import React from 'react';
+import React, { Component } from 'react';
 import './App.scss';
+import { Clock } from './Clock';
 
-function getRandomName(): string {
-  const value = Date.now().toString().slice(-4);
+type State = {
+  hasClock: boolean
+  clockName: string;
+};
 
-  return `Clock-${value}`;
-}
+export class App extends Component<{}, State> {
```

Comment:
Better create an interface for props even if it's an empty object (it'll be easier to update in future)

Reply:
Today you don’t have any props, but tomorrow you will start receiving some props )
It will be easier to update your component if interfaces are prepared